tests/e2e/tutorials/isolve-gpu.js

```diff
@@ -37,19 +36,17 @@ async function runTutorial() {
       "output.h5",
       "log.tgz"
     ];
-    await tutorial.checkNodeResults(1, outFiles);
-
-    await tutorial.toDashboard();
-
-    await tutorial.removeStudy(studyId);
+    await tutorial.openNodeFiles(1)
+    await tutorial.checkResults2(outFiles);
   }
   catch(err) {
     tutorial.setTutorialFailed(true);
     console.log('Tutorial error: ' + err);
   }
   finally {
+    await tutorial.toDashboard();
+    await tutorial.removeStudy2(studyId);
```

Comment:
👍

Reply:
I see the purpose of removeStudy2. Why don't you rename it and move the call to the ```catch``` section?

Reply:
agree with @odeimaiz, then we could detect when the remove study fails and still remove the studies